slack_sdk/socket_mode/builtin/connection.py

```diff
@@ -250,7 +250,7 @@ def check_state(self) -> None:
                     is_stale = disconnected_seconds > self.ping_interval * 4
                     if is_stale:
                         self.logger.info(
-                            "The connection seems to be stale. Disconnecting..."
+                            "The connection seems to be stale. Reconnecting..."
```

Comment:
Just to clarify and for my own understanding: so the next `disconnect()` call will end up actually re-establishing the connection via the monitoring loop you added?

Reply:
Thanks. You're right. This change should be reverted as the situation here is not always auto-reconnection enabled.